docs/earth-mdt.rst

```diff
@@ -23,7 +23,7 @@ for pixel-registered grids; gridline-registered grids increment dimensions by on
 
 .. _tbl-earth_mdt:
 
-.. table:: CNES Global Earth Mean Dynamic Topography. An asterisk denotes tiled datasets.
+.. table:: CNES Earth Mean Dynamic Topography. An asterisk denotes tiled datasets.
```

Comment:
This could be deleted. There is no tiled datasets for mdt.

Reply:
This is updated in commit https://github.com/GenericMappingTools/remote-datasets/pull/108/commits/9bca7e7a5e20a42c7455de2e001eb77d74bdf724.